requirements.txt

```diff
@@ -26,13 +26,13 @@ typing-extensions==3.10.0.0
 urllib3==1.26.5
 django-markdownify==0.9.1
 # Octopoes-core, specify a tagged version
-git+https://github.com/minvws/nl-kat-octopoes@v1.1.0
-celery==5.2.2
-flower==1.0.0
+git+https://git@github.com/minvws/nl-kat-octopoes@v1.2.0
```

Comment:
User Git is not needed.
Also, this url gives a 404

Reply:
Fixed in https://github.com/minvws/nl-kat-rocky/pull/28